data/libraries.json

```diff
@@ -112,5 +112,13 @@
         "web_components": false,
         "cdn_url": "https://cdn.jsdelivr.net/gh/hbroek/reken/dist/reken.min.js",
         "categories": ["ui"]
+    },
+    {
+        "name": "turbo",
+        "repo_url": "https://github.com/hotwired/turbo",
+        "size": "75 KB",
```

Comment:
I believe they're using gzip sizes which would be ~19kb

https://bundlephobia.com/package/@hotwired/turbo@7.3.0

```suggestion
        "size": "~19 KB",
```

![BF314341-0AAC-4AFC-9610-14C89A84DA59](https://github.com/adamghill/unsuckjs.com/assets/26425882/03eda660-7a34-4108-be66-9673d3423939)